pepper-apis/src/main/java/org/broadinstitute/ddp/route/PostPasswordResetRoute.java

```diff
@@ -34,16 +34,17 @@
     @Override
     public Object handle(Request request, Response response) throws Exception {
         String auth0ClientId = request.queryParams(QueryParam.AUTH0_CLIENT_ID);
+        String auth0Domain = request.queryParams(QueryParam.AUTH0_DOMAIN);
```

Comment:
API doc changes here as well please.  Maybe a utility query in `JdbiClient` to do the query-by-client-and-see-if-you-get-just-one thing?

Reply:
Question: who is passing in this `domain` query param? It looks like this might be Auth0 calling back to the server? Maybe you need to update auth0 email configs? (look in `study-builder/tenants/cmi/emails/change_password.html`).

Reply:
I implemented the approach suggested by @zyme: if a domain is missing, we check if the clientId is unique. If it is, we proceed or else fail. If the domain is present, we use it. From the client's point of view, there is no difference until we start getting clientId clashes.